autoresearch/scripts/run_experiment.py

```
@@ -0,0 +1,317 @@
+            args.overrides,
+            tier,
+            {"error": "no output dir found"},
+        )
+        sys.exit(1)
+
+    print(f"Output directory: {output_dir}")
+
+    # Run evaluator
+    evaluator_path = PROJECT_ROOT / "autoresearch" / "evaluators" / "labelfree_evaluator.py"
+    try:
+        eval_result = subprocess.run(  # nosec B603
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)